src/System.IO/ref/System.IO.cs

```diff
@@ -162,13 +170,17 @@ public abstract partial class Stream : System.IDisposable
         public abstract long Position { get; set; }
         public virtual int ReadTimeout { get { return default(int); } set { } }
         public virtual int WriteTimeout { get { return default(int); } set { } }
+        public virtual System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { return default(System.IAsyncResult); }
```

Comment:
Could all of these be marked `[EditorBrowsableAttribute(EditorBrowsableState.Never)]` in the contract to hide them from intellisense, or would that be confusing to devs who need to override them?